crds/fed.solo.io_v1_crds.yaml

```diff
@@ -1,46 +0,0 @@
-# Code generated by skv2. DO NOT EDIT.
-
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  labels:
-    app: ""
-    app.kubernetes.io/name: ""
-  name: glooinstances.fed.solo.io
-spec:
-  group: fed.solo.io
-  names:
-    kind: GlooInstance
-    listKind: GlooInstanceList
-    plural: glooinstances
-    singular: glooinstance
-  scope: Namespaced
-  subresources:
-    status: {}
-  versions:
-  - name: v1
-    served: true
-    storage: true
-
----
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
note to future reviewers: we don't care about the deletion of the failoverscheme crd because gloo (the only user of the `gloo-repo-branch` branch) doesn't reference the crds from solo-apis